calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx

```diff
@@ -0,0 +1,9 @@
+DD March 2024
+
+### Improvements
+-
```

Comment:
- Decrease the validity of JWTs issued by Dex to 15m (down from 24h).

@mazdakn @sridhartigera Do we need a release note for the CRD changes?

Reply:
- Policy recommendation excludes Openshift namespaces by default.

Reply:
- If kube-controllers metric port is set to 0, no ingress rule will be created.

Reply:
Added all.